### DISCUSSION TITLE
All Molten Editor commands don't show up and don't work

### DISCUSSION THREAD

**Discussion by zorjo** [benlubas/molten-nvim#199] [Q&A] [answered]
Error E492: Not an editor command
Even autocomplete to any `:Molten*` is not working currently.
It used to work, a month ago. 
Environment is Windows 10. 
Config of plugin is 
```
  {
    "benlubas/molten-nvim",
    version = "^1.0.0", -- use version <2.0.0 to avoid breaking changes
    build = ":UpdateRemotePlugins",
    init = function()
      vim.g.molten_auto_open_output = false
      vim.g.molten_output_show_more = true
      vim.g.molten_output_virt_lines = true
      vim.g.molten_split_direction = "right" --direction of the output window, options are "right", "left", "top", "bottom"
      vim.g.molten_split_size = 40 --(0-100) % size of the screen dedicated to the output window
      vim.g.molten_virt_text_output = true
      vim.g.molten_use_border_highlights = true
      vim.g.molten_virt_lines_off_by_1 = true
      vim.g.molten_auto_image_popup = false
    end,
  },
}
```
rplugin after `:UpdateRemotePlugins`
```
" perl plugins


" node plugins


" python3 plugins


" ruby plugins


" python plugins


```

**Comment by benlubas** [comment]
Run `:UpdateRemotePlugins` manually. Lazy just bugs out and doesn't do it sometimes

**Reply by zorjo** [reply]
I ran it, got the message 
```
remote/host: python3 host registered plugins []
remote/host: generated rplugin manifest: C:\Users\ASUS\AppData\Local\nvim-data/rplugin.vim
```
Restarted nvim, nothing changed.
I hope this isn't just me

**Reply by benlubas** [reply, marked as answer]
if you're on windows you have to read [Windows.md](https://github.com/benlubas/molten-nvim/blob/main/docs/Windows.md)

**Reply by zorjo** [reply]
Thanks, I should have realized earlier. Don't know why the rplugin.vim file was overwritten. Thanks anyway!

**Reply by benlubas** [reply]
windows remote plugin bug. been around for a while, but barely anyone that works on nvim uses windows so y'all are kinda left out to dry :/

I'm pretty sure there's a remote plugin refactor in the works, so hopefully that will address this issue too
